src/main/scala/org/holmesprocessing/totem/services/pemeta/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM golang:alpine
+
+# Create folder
+RUN mkdir -p /service
+WORKDIR /service
+
+# Get Golang and PEV dependencies
+RUN apk add --no-cache \
+        git \
+        && go get github.com/julienschmidt/httprouter \
+        && git clone -b testing https://github.com/boddumanohar/libpe.git \
```

Comment:
Yes. They reviewed yesterday and asked for changes in design (i.e move each functionality to seperate files). Today, for some time I will work on that. If that is merged, then I will make an update pull request with orginal pev as dependency.